### PR TITLE
PRS IDs from Tanasee 1 - no. 3

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
New PRS IDs from Tanasee1 as discussed in pull request #742 and revised by Dorothea.

### CODE REVIEW

File: new/PRS14049GabraMasqal.xml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+            <change who="CH" when="2023-07-21">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ገብረ፡ መስቀል፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Gabra Masqal</persName>
+                    <persName><roleName type="title" xml:lang="gez">አቡነ፡</roleName></persName>
```
Comment (eu-genia) [inline new/PRS14049GabraMasqal.xml:50]: in the first batch and below titles all transcribed ?

File: new/PRS14065Tewogolos.xml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+            <change who="CH" when="2023-08-24">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ቴዎጎሎስ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Tewogolos</persName>
+                    <occupation type="political" when="1507">ʿaqābe saʿat</occupation>
```
Comment (eu-genia) [inline new/PRS14065Tewogolos.xml:50]: ʿaqqābe ?

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/PRS14049GabraMasqal.xml (hunk 1 above)
Comment (CarstenHoffmannMarburg) [reply new/PRS14049GabraMasqal.xml:50]: I transcribed all the titles in each of the the records.

File: new/PRS14065Tewogolos.xml (hunk 2 above)
Comment (CarstenHoffmannMarburg) [reply new/PRS14065Tewogolos.xml:50]: Done.

**Review by thea-m — APPROVED** [review]

**Pull request merged**